charts/nginx-ingress-services/values.yaml

```diff
@@ -40,6 +40,9 @@ certManager:
   #  (1) required to be set in values.yaml
   #  (2) used by Letsencrypt to send expiration notices and alike
   certmasterEmail:
+  # can be used to use custom solvers, such as dns01
+  # you may use secrets.certManager.additionalSolvers to store secrets
+  additionalSolvers:
```

Comment:
This doesn't explain what structure / kind of resource(s) we expect here.

Plural suggest its multiple. List or map, `Issuer`/`ClusterIssuer`?